rfcs/taskforce/0004-rfc-process-update.md

```diff
@@ -0,0 +1,79 @@
+- When you are ready to submit your RFC, create a PR if it hasn't already been done (make sure to remove the [DRAFT] in the title) and request a reviews from the appropriate team or team members.
+- Discuss and incorporate feedback - This is an active process, with the goal to resolve and RFC withing a week after submitting for review, unless substantial changes are required.  The owner / champion should drive this process, making sure to follow up with the team members whose approval is required to get the necessary reviews.  To facilitate this process:
+    - Make sure that the PRs are added to the appropriate team board either in the discuss column or the review column so outstanding RFCs can be highlighted during the team meetings
+    - Add it to the list to be discussed at the engineering all hands
+    - If there is a lot of active discussion or not enough discussion, call a meeting with the goal of making a decision. If there’s little or no concern then reviewers should go ahead and approve. Then move forward with the proposed solution.
+
+**RFC Life Cycle**
+
+- Draft - Started but not yet ready for review.  Marked with [DRAFT] in title.
+- Open - Ready for review with active discussion and incorporation of feedback.
+- Merged - When feedback has been addressed and the feature is ready to be implemented the RFC can be merged.  If there are outstanding questions make sure to capture them in the RFC as well.
+- Deprioritzed RFCs - Mark them as postponed as close them. Reopen when they are relevant again.  Mark with the label "Postponed RFC"
+- Abandoned RFCs - RFCs that we decide not to do should be closed.
+
+**CleanUp Stale RFCs**
+
+As part of exiting taskforce, have a effort to close out stale RFCs either closing them as Postponed/Abandoned or driving them to decision and landing them.   
+
+
+**Updating RFCs** 
+
+In general an RFC's purpose is to gain consensus on an approach and act as an artifact for technical design decisions.  It is not a living document of how the code works.  Still there may be times when an RFC should be updated
+
+- Addressing open or unresolved questions
+- Revisiting and updating the approach if it has been changed between the RFC being merged and the
+
+RFCs should not be updated to reflect actual implementation. That should be captured in docs and done as part of the development process (as captured in the Definition of Done - TBA)
+
+**The RFC Template**
+
+Suggest adding the following section(s) to the RFC Template:
+
+- Testing - discussing how this feature or changes should be tested and if there's anything (eg. infrastructure) that needs to be done to support that
+
+
+## Unresolved questions
+[unresolved-questions]: #unresolved-questions
+
+- Should we RFC substantial changes to our processes as well? Might be good for visibility.
+- Should we have a code owners team for the rfc folder. Leaning towards no but worth discussing
```

Comment:
We used to and this slowed us down a lot, but see my comment above -- we should set clear expectations around who and how many approvals are needed.